design/one-pager-stack-relationship-labels.md

```diff
@@ -0,0 +1,229 @@
+# Stack Parent/Child Relationship Label Spec
+
+- Owner: Steven Rathbauer ([@rathpc](https://github.com/rathpc))
+- Reviewers: Crossplane Maintainers
+- Status: Draft
```

Comment:
@rathpc can you add a note for clarity that this may not be implemented right away and it's potentially focused on longer term?